src/components/ServerList.tsx

```
@@ -103,1052 +115,1073 @@ import { Skeleton } from "./ui/skeleton";
+									updatedSL[b.server] = b.motd;
+								});
+								setMotdList(updatedSL);
+								setServers(serverList.currentServers);
+								setLoading(false);
+							});
+						}}
+						loader={<LoadingSpinner className="flex items-center" />}
+						endMessage={
+							<p
+								style={{ textAlign: "center" }}
+								dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)